docs/FRSKY_R9.md

```diff
@@ -54,24 +54,30 @@ If you have the older 2018, "ACCST" version of the R9M, you will want to perform
 
 ### DIY Inverter Dongle ###
 
-There are several DIY approaches for building an inverter dongle. A common approach is based on the MAX3232 RS232 chip; an excellent build tutorial is provided here [Some soldering required](https://discuss.ardupilot.org/t/some-soldering-required/27613). Be aware however that many fake chips are available, and the original scheme suggested in this blog may not work. One, in fact, may have to two extra diodes and a resistor.
+There are several DIY approaches for building an inverter dongle. A common approach is based on the MAX3232 RS232 chip; an excellent build tutorial is provided here [Some soldering required](https://discuss.ardupilot.org/t/some-soldering-required/27613). Be aware however that many fake chips are available, and the original scheme suggested in this blog may not work. One, in fact, may have to add two extra diodes and a resistor.
 
 <img src="images/frsky-max3232-inverter-scheme.jpg" width="360px">
 
 ### ESP32 Wireless Bridge ###
 
-The mLRS git repository includes an Arduino sketch which allows several supported ESP32 boards to be used as a wireless bridge to connect the serial port to any of the many available GCS such as Mission Planner or QGroundControl. This approach can also eliminate the need for a separate inverter dongle.  Two of these boards, the M5Stamp Pico Mate and the M5Stamp C3U Mate from M5Stack allow pin layouts which are especially convenient to connect directly to the serial pins on the R9M Tx module. The [M5Stamp C3U Mate](https://shop.m5stack.com/collections/m5-controllers/products/m5stamp-c3u-mate-with-pin-headers) is the easiest option as it can be flashed via its included USB port rather than requiring a separate programmer. The 2.4 GHz wireless bridge works especially well with 900 MHz systems like the R9 since the separate frequency range minimizes interference.
+The mLRS git repository includes an Arduino sketch which allows several supported ESP32 boards to be used as a wireless bridge to connect the serial port to any of the many available GCS such as Mission Planner or QGroundControl. This approach can also eliminate the need for a separate inverter dongle.  Two of these boards, the M5Stamp Pico Mate and the M5Stamp C3U Mate from M5Stack allow pin layouts which are especially convenient to connect directly to the serial pins on the R9M Tx module. The [M5Stamp C3U Mate](https://shop.m5stack.com/collections/m5-controllers/products/m5stamp-c3u-mate-with-pin-headers) is the easiest option as it can be flashed via its included USB port rather than requiring a separate programmer, but it doesn't support the Bluetooth Serial Port Profile.  The M5Stamp Pico Mate is the best choice if you might want to use Bluetooth serial.  Both of these 2.4 GHz wireless bridge options work especially well with 900 MHz systems like the R9 since the separate frequency range minimizes interference.
```

Comment:
Suggest a small rewrite:

'The [M5Stamp C3U Mate](https://shop.m5stack.com/collections/m5-controllers/products/m5stamp-c3u-mate-with-pin-headers) is the easiest option as it can be flashed via its included USB port and does not require a separate programmer.  However, it doesn't support Bluetooth Classic. If you want to use Bluetooth Classic then the M5Stamp Pico Mate is the recommended choice.  Both of these modules use the 2.4 GHz band for the wireless bridge and will work well with 868/915 MHz systems like the R9 as the separate frequency range minimizes interference.'

Reply:
Yes, that is a little better

Reply:
- maybe replace "doesn't" by "does not"
- I think the M5Stamp Pico Mate could also be a link

Reply:
Resolved in latest commit